examples/durable/DurableApp/MonitorOrchestrator/run.ps1

```diff
@@ -0,0 +1,26 @@
+using namespace System.Net
+
+param($Context)
+
+Write-Host 'MonitorOrchestrator: started.'
+
+$jobId = $Context.Input.JobId
+$pollingInterval = $Context.Input.PollingInterval
+$expiryTime = $Context.Input.ExpiryTime
+
+while ($Context.CurrentUtcDateTime -lt $expiryTime) {
+    $jobStatus = Invoke-ActivityFunction -FunctionName 'GetJobStatus' -Input $jobId
+    if ($jobStatus -eq "Completed") {
+        # Perform an action when a condition is met.
+        $output = Invoke-ActivityFunction -FunctionName 'SendAlert' -Input $machineId
```

Comment:
Where is `$machineId` from?

Reply:
I believe the user supplies this value, as it is not passed into the orchestrator in either the C# or JavaScript template.